RFS/docs/DESIGN.md

```diff
@@ -222,25 +214,18 @@ If no Entry is returned, we know that this sub-phase is complete and attempt to
 
 The work lease for this sub-phase is on the Shard (ensuring every Elasticsearch Document in that Shard has been processed).  We log/emit metrics to indicate how many Documents are successfully and unsuccessfully migrated but we don’t consider the Shard Work Entry to have failed if some (or even all) of the Documents in it are unsuccessfully migrated.  We only retry the Shard Work Entry when an RFS Worker fails to process every Document within the lease window.  These retries are relatively time consuming, but safe because we overwrite any partial work performed by a previous RFS Worker. 
 
-
 ## Appendix: Assumptions
 
 We start with the following high-level assumptions about the structure of the solution.  Changes to these assumptions would likely have a substantial impact on the design.  
 
 * (A1) - The RFS Workers cannot assume access to a data store other than the migration’s target cluster as a state-store for coordinating their work.
 * (A2) - The RFS Worker will perform all the work required to complete a historical migration.
-* (A3) - The RFS Scaler will only need to scale one type of Docker composition, which may be composed of multiple Containers, one of which is the RFS Worker
 
 We have the following, additional assumptions about the process of performing a Reindex-from-Snapshot operation:
 
-* (A4) - The creation of Elasticsearch Templates is atomic
-* (A5) - The creation of Elasticsearch Indices is atomic
-* (A6) - Re-doing portions of the overall RFS operation is fine, as long as every portion is completed at least once
-* (A7) - Elasticsearch Templates must be migrated in the order: first Legacy, then Composite, then Index
-* (A8) - All Elasticsearch Templates must be migrated before any Elasticsearch Indices can be migrated
-* (A9) - Elasticsearch Indices can be migrated in parallel without ordering concerns
-* (A10) - Elasticsearch Shards can only be migrated after their Elasticsearch Index has been migrated
-* (A11) - Elasticsearch Shards can be migrated in parallel without ordering concerns
+* (A3) - Re-doing portions of the overall RFS operation is fine, as long as every portion is completed at least once
+* (A4) - Elasticsearch Shards can only be migrated after their Elasticsearch Index has been migrated
```

Comment:
Given the discussions about defaults and manually setting metadata, this could also be "migrated or created"